prow/cmd/deck/static/script.js

```diff
@@ -94,7 +94,16 @@ function groupKey(build) {
     return build.repo + " " + build.number + " " + build.refs;
 }
 
+function configure() {
```

Comment:
I think you need to call this function from each of the `*script.js` files.

Reply:
Yeap, still WIP.